packages/teleterm/src/ui/QuickInput/useQuickInput.ts

```diff
@@ -46,8 +46,8 @@ export default function useQuickInput() {
     setActiveSuggestion(index);
   };
 
-  const onPickSuggestion = (index: number) => {
-    if (!hasSuggestions) {
+  const onEnter = (index?: number) => {
```

Comment:
Renamed it from `onPickSuggestion` to `onEnter`, because it's no longer about picking a suggestion, but kinda doing the next step in the command bar.

I wanted to have a name that isn't really specific to a key that's being used to perform the action. So initially I thought about `onProceed`, but then I figured out I may as well use `onEnter`. Let me know if you have a better idea for the name.